### DISCUSSION TITLE
Issue in running

### DISCUSSION THREAD

**Discussion by shubhhtri** [dcm4che/dcm4che#1317] [General]
I am a newbie in Java. I have installed every requirement of the project and also build it. I am unable to run the project on my local machine. Please share any resources which could help me.

**Comment by hczedik** [comment]
If you just want to run the dcm4che tools, then download the "Binary Distribution Packages" (click the link on https://github.com/dcm4che/dcm4che/releases ), extract the `...-bin.zip` file, and then run any of the tools within the "bin" directly from the command line.
